@here/olp-sdk-dataservice-write/README.md

```diff
@@ -1,5 +1,253 @@
 # DataService Write
 
+## Overview
+
+This repository contains the complete source code for the HERE Data SDK for TypeScript Dataservice Write `@here/olp-sdk-dataservice-write` project. `olp-sdk-dataservice-write` is a TypeScript library that will upload data to the HERE platform catalogs.
+
+# Publish data
+
+You can use the HERE Data SDK for TypeScript to publish data to catalogs.
+Now, we only support publishing data to a versioned layer.
+
+You can publish data to a [versioned layer](https://developer.here.com/olp/documentation/data-user-guide/portal/layers/layers.html#versioned-layers), if you need to store and access the history of previous data updates. To achieve consistency between layers, publish any update that affects multiple versioned layers together in one publication. You can access a new catalog version when all layers are updated, and the publication is finalized. Once you publish a version, you cannot change the data in that version and can [remove](#cancel) it only by removing the whole catalog version.
+
+The maximum number of partitions you can upload and publish in one request is 1,000. If you have more than 1,000 partitions, upload the data and metadata for the first 1,000 partitions, and then do the same for the next set of partitions.
+
+**To publish data to the versioned layer:**
+
+1. Import data from the `olp-sdk-dataservice-write` module.
+
+    Add to your dependencies:
+
+    ````
+    "@here/olp-sdk-dataservice-api": "1.5.0",
+    "@here/olp-sdk-dataservice-write": "1.0.0"
+    ```
```

Comment:
Make it step 1.

```
1. Add the `@here/olp-sdk-dataservice-api` and `@here/olp-sdk-dataservice-write` to your dependencies.
    ````
    "@here/olp-sdk-dataservice-api": "1.5.0",
    "@here/olp-sdk-dataservice-write": "1.0.0"
    ```
2. Import data from the `olp-sdk-dataservice-write` module.
    ```typescript
       import {
         VersionedLayerClient,
         StartBatchRequest,
         CompleteBatchRequest,
         UploadPartitionsRequest,
         UploadBlobRequest,
       } from "@here/olp-sdk-dataservice-write";
       ```
```